vision/beta/cloud-client/src/main/java/com/example/vision/DetectBatchAnnotateFiles.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.vision;
+
+// [START vision_detect_batch_annotate_files_beta]
+
+import com.google.api.core.ApiFuture;
+import com.google.cloud.vision.v1p4beta1.AnnotateFileRequest;
+import com.google.cloud.vision.v1p4beta1.AnnotateFileResponse;
+import com.google.cloud.vision.v1p4beta1.BatchAnnotateFilesRequest;
+import com.google.cloud.vision.v1p4beta1.BatchAnnotateFilesResponse;
+import com.google.cloud.vision.v1p4beta1.Block;
+import com.google.cloud.vision.v1p4beta1.Feature;
+import com.google.cloud.vision.v1p4beta1.Feature.Type;
+import com.google.cloud.vision.v1p4beta1.ImageAnnotatorClient;
+import com.google.cloud.vision.v1p4beta1.InputConfig;
+import com.google.cloud.vision.v1p4beta1.Page;
+import com.google.cloud.vision.v1p4beta1.Paragraph;
+import com.google.cloud.vision.v1p4beta1.Symbol;
+import com.google.cloud.vision.v1p4beta1.TextAnnotation;
+import com.google.cloud.vision.v1p4beta1.Word;
+import com.google.protobuf.ByteString;
+
+import java.io.FileInputStream;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+
+public class DetectBatchAnnotateFiles {
+
+  /**
+   * Performs document feature detection on a local PDF/TIFF/GIF file.
+   * While your PDF file may have several pages, this API can process up to 5 pages only.
+   * @param filePath The local file path e.g. "path/to/file.pdf"
+   * @throws Exception on errors
+   */
+  public static void detectBatchAnnotateFiles(String filePath) {
+
+    // Initialize the client that will be used to send requests. This client only needs to be
+    // created
+    // once, and can be reused for multiple requests. After completing all of your requests, call
+    // the "close" method on the client to safely clean up any remaining background resources.
```

Comment:
I think we can remove this comment.

Reply:
Fixed across all files in latest commit.